wagtailio/project_styleguide/templates/patterns/base_page.html

```diff
@@ -1,5 +1,5 @@
 {% extends "patterns/base.html" %}
-{% load static wagtailcore_tags wagtailimages_tags wagtail_ab_testing_tags wagtailio_utils %}
+{% load static wagtailcore_tags wagtailimages_tags wagtail_ab_testing_tags wagtailio_utils navigation_tags %}
```

Comment:
Question: What are we adding this template tag for here? I don't think there's a template tag below that isn't in it's own template.

Reply:
Good point, removed in 25cd9d15